tests/conftest.py

```diff
@@ -12,7 +12,7 @@
 
 HADOOP_VERSION = '2.8.5'
 SPARK_VERSION = '2.4.4'
-SPARK_GIT_COMMIT = 'c3e32bf06c35ba2580d46150923abfa795b4446a'  # 2.4.3
+SPARK_GIT_COMMIT = '7955b3962ac46b89564e0613db7bea98a1478bf2'  # 2.4.4
```

Comment:
Did you need to make this change to get tests to pass? I'd keep it out of the PR since it's unrelated to the topic at hand.